samples/ContainmentSuperVsExtendsSameType.java

```diff
@@ -20,7 +20,7 @@
 class ContainmentSuperVsExtendsSameType {
   void x() {
     // :: error: jspecify_nullness_mismatch jspecify_but_expect_nothing
-    new Check<Lib<? extends Object>, Lib<? super Object>>();
+    new Check<Lib<? extends Number>, Lib<? super Number>>();
```

Comment:
I had typed a comment here, but I don't know what happened to it....

My comment was that my guess was that https://github.com/jspecify/jspecify/commit/6f78b15de1bcf1dd7866a7663b88875cc756faac was describing a problem specific to `Object`. If so, then it would be nice to use `jspecify_conflicting_annotations` here instead of changing the type. (Or we could have that _plus_ this :))

It occurs to me now, though, that if the tests don't fail, then I'm wrong in claiming that the problem is specific to `Object` :)

Reply:
There are still test cases for `? super Object` - the point here I think is something else.
There is an `jspecify_but_expect_nothing` here, so I will look later why no error is raised.